webapp/components/post_view/components/post_list.jsx

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) 2015 Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information.
 
+import _ from 'lodash';
```

Comment:
Where is this library coming from? I don't see an entry in `package.json`. Also I don't really see the value in adding a big library just for this.